src/features/profile/editProfileForm/model/lib/validation/editProfileSchema.ts

```diff
@@ -18,13 +18,7 @@ export const editProfileSchema = yup.object().shape({
 	specialization: yup
 		.number()
 		.required(i18n.t(Validation.SPECIALIZATION_REQUIRED, { ns: i18Namespace.validation })),
-	phone: yup
-		.string()
-		.required(i18n.t(Validation.REQUIRED, { ns: i18Namespace.validation }))
-		.matches(
-			/^\+7-\(\d{3}\)-\d{3}-\d{2}-\d{2}$/,
-			i18n.t(Validation.PHONE, { ns: i18Namespace.validation }),
-		),
+	phone: yup.string().nullable(),
```

Comment:
Вот тут не нужно удалять валидацию. Так как если телефон будут вводить, смогут ввести что угодно.
Почитай документацию yup, можно сделать проверку, чтобы проверять валидацию только при наличии значения в телефоне 
Вот пример, чтобы ты понимал что я имею ввиду:
```
count: number().when('isBig', {
    is: (val) => val == true,
    then: (schema) => schema.min(5),
    otherwise: (schema) => schema.min(0),
  }),
```